src/leafdoc.js

```diff
@@ -471,7 +471,7 @@ Leafdoc.prototype.addStr = function(str, isSource) {
 				} else if (directive === 'param') {
 					// Params are param name, type.
 					/// TODO: Think about default values, or param explanation.
-					var split = content.split(',');
+					var split = content.split(':');
```

Comment:
You're right. It's been quite a long time since I split the `@param` directive and the shorthand syntax.